src/quo2/components/avatars/wallet_user_avatar/component_spec.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.avatars.wallet-user-avatar.component-spec
+  (:require [quo2.components.avatars.wallet-user-avatar.view :as wallet-user-avatar]
+            [test-helpers.component :as h]))
+
+(h/describe "wallet user avatar"
+  (h/describe "View internal"
+    ;; An example of a clearer test description
+    (h/test "Renders by default even with no input parameters"
+      (h/render
+       [wallet-user-avatar/wallet-user-avatar {}])
+      (h/is-truthy (h/query-by-label-text :wallet-user-avatar)))
+
+    ;; If we decide not to test styles but still want to test other props:
```

Comment:
nit: personally I prefer to self-comment on my prs rather than add comments like this in the code which I am intending to remove later 👍